code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -639,6 +639,136 @@
 	w_class = ITEM_SIZE_NORMAL
 	price_tag = 500
 
+/obj/item/tool/cannibal_scythe
+	name = "\"Cannibal Strike\" scythe"
+	desc = "A heavy grotesque handmade scythe made from parts of the most terrible creatures living on Amethyn. A real hunter's weapon that requires serious skills to handle itself.\
+	 What are you waiting for, hunter? The Matriarch is calling for a great hunt!"
```

Comment:
Cool desc but maybe since the matriarch is gone just refer to the great hunt calling so that it doesn't refer to a long since gone figure